README.md

```diff
@@ -13,10 +11,23 @@ This plugin can be enabled with the following configuration in your `settings.js
   ...
   "plugins": {
     ...
-    "csp-simple-bodies": {
+    "csp-simple-wms-bodies": {
       "bodies": {
         <anchor name>: {
-          "texture": <path to surface texture>
+          "texture": <string>,        // The path to surface texture.
```

Comment:
Write a comment that this texture is used as background texture. The texture from the WMS will be overlaid